docs/lib/auth/fragments/ios/signin_next_steps/10_signin.md

```diff
@@ -0,0 +1,17 @@
+
+The signin apis if it succeeds you will get back an associated value of type `AuthSignInResult`, and you can access the next step through the property `nextStep` of `AuthSignInResult`. 
+
+```swift
+func signIn(username: String, password: String) {
+    Amplify.Auth.signIn(username: username, password: password) { result in
+        switch result {
+        case .success(signinResult):
+            print("Next step = ")
```

Comment:
This example code is missing a critical part, no?
```suggestion
            print("Next step = \(signInResult.nextStep)")
```
Other thought: does it make sense to `switch` over the various possible next steps, here?

Reply:
Changed to switch statement...